main.go

```diff
@@ -184,10 +183,7 @@ func main() {
 						default:
 							event, err := cur.Watch(ctx)
 							if err != nil {
-								if errors.Is(err, watcher.ErrWatchTimeout) {
-									continue
-								}
-								logger.Error(err, "Error monitoring watcher interface.")
```

Comment:
Should we keep this logger call?

Reply:
I decided to remove it because putting it before the `continue` makes `enduro` very noisy printing each couple of seconds:

```
replaceafill@eluk:~/dev/archivematica/enduro$ make run
mkdir -p ./build
go build -trimpath -o build/enduro -ldflags='-X "main.buildTime=2020-04-30T02:00:04+0000" -X main.gitCommit=86620c5' -v
./build/enduro
2020-04-29T20:00:07.822-0600	INFO	enduro	enduro@/main.go:96	Starting...	{"version": "(dev-version)", "pid": 6155}
2020-04-29T20:00:07.822-0600	INFO	enduro	enduro@/main.go:100	Configuration file loaded.	{"path": "/home/replaceafill/dev/archivematica/enduro/enduro.toml"}
2020-04-29T20:00:07.922-0600	INFO	enduro.cadence-worker	enduro@/main.go:239	Started Worker	{"Domain": "enduro", "TaskList": "global", "WorkerID": "6155@eluk@global"}
2020-04-29T20:00:09.887-0600	ERROR	enduro	enduro@/main.go:186	Error monitoring watcher interface.	{"error": "watcher timed out"}
github.com/go-logr/zapr.(*zapLogger).Error
	github.com/go-logr/zapr@v0.1.1/zapr.go:128
main.main.func4
	github.com/artefactual-labs/enduro@/main.go:186
github.com/oklog/run.(*Group).Run.func1
	github.com/oklog/run@v1.1.0/group.go:38
2020-04-29T20:00:11.894-0600	ERROR	enduro	enduro@/main.go:186	Error monitoring watcher interface.	{"error": "watcher timed out"}
github.com/go-logr/zapr.(*zapLogger).Error
	github.com/go-logr/zapr@v0.1.1/zapr.go:128
main.main.func4
	github.com/artefactual-labs/enduro@/main.go:186
github.com/oklog/run.(*Group).Run.func1
	github.com/oklog/run@v1.1.0/group.go:38
2020-04-29T20:00:13.903-0600	ERROR	enduro	enduro@/main.go:186	Error monitoring watcher interface.	{"error": "watcher timed out"}
github.com/go-logr/zapr.(*zapLogger).Error
	github.com/go-logr/zapr@v0.1.1/zapr.go:128
main.main.func4
	github.com/artefactual-labs/enduro@/main.go:186
github.com/oklog/run.(*Group).Run.func1
	github.com/oklog/run@v1.1.0/group.go:38
2020-04-29T20:00:15.910-0600	ERROR	enduro	enduro@/main.go:186	Error monitoring watcher interface.	{"error": "watcher timed out"}
github.com/go-logr/zapr.(*zapLogger).Error
	github.com/go-logr/zapr@v0.1.1/zapr.go:128
main.main.func4
	github.com/artefactual-labs/enduro@/main.go:186
github.com/oklog/run.(*Group).Run.func1
	github.com/oklog/run@v1.1.0/group.go:38
```

Reply:
It we hid all errors then the operator won't be able to tell what's going on, would the following work?

```go
event, err := cur.Watch(ctx)
if err != nil {
	if !errors.Is(err, watcher.ErrWatchTimeout) {
		logger.Error(err, "Error monitoring watcher interface.")
	}
	continue
}
```

Reply:
Thanks!